### PR TITLE
Mhp 2348 updates to css for web

### PR DESCRIPTION
**Pull request by jesusfreak3**

### CODE REVIEW

**Comment by jesusfreak3** [comment]
@OzzieOrca its weird you were not seeing my changes.  Going to take a look

**Comment by jesusfreak3** [comment]
@OzzieOrca ok i think it should be good now.

File: app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-                <img class="db mr1" ng-src="{{$ctrl.chevronLeftIcon}}"/>
-            </a>
-            {{$ctrl.survey.title}}
-    </div>
+<div class='br3 shadow-3 mb4 overflow-hidden'>
+    <div class="header flex justify-between">
+        <div class="">
+            <div class="f3 pivot-dark-blue ph3 pt3 flex">
+                <a  ui-sref="app.ministries.ministry.surveys({ surveyId: survey.id })">
+                    <span class='pivot-fill-dark-blue mr-2'><</span>
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html]: This is killing Netlify. Use the HTML entity :)

File: app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html [hunk 2]
```diff
@@ -1,13 +1,14 @@
-<div class="header">
-    <div class="flex justify-between items-center pv3 ph2">
-        <div class="survey-title">
+<div class="br3 shadow-3 mb4 overflow-hidden">
+    <div class="flex justify-between items-center pv3 ph2 bg-white">
+        <div class="f3 pivot-dark-blue ph3 flex">
             <a ui-sref="app.ministries.ministry.surveyResponses({ surveyId: survey.id })">
-                <img class="db mr1" ng-src="{{$ctrl.chevronLeftIcon}}"/>
+                <span class='pivot-fill-dark-blue mr-2'><</span>
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html]: Another one

Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html]: I keep forgetting about that issue with netlify~

Comment (OzzieOrca) [reply app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html]: Idk why it's not an issue locally. Maybe a different version of node or something...

File: app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html (hunk 1 above)
Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html]: fixed

File: app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html (hunk 2 above)
Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/surveyResponses/surveyResponses.html]: fixed

**Comment by jesusfreak3** [comment]
> Tools menu still getting cut off if the container isn't big enough:
> screen shot 2019-02-20 at 1 01 11 pm
> 
> Also tools menu doesn't seem to open on hover or click on root of ministries tab
> screen shot 2019-02-20 at 1 02 38 pm

Right, never tested that with no results, overflow hidden is set to keep the silly roudned border at the bottom corners since there are backgrounds that overflow in the results... Ill adjust.

**Comment by OzzieOrca** [comment]
Oh I guess both my screenshots are the same root overflow issue. Didn't realize that :)

**Comment by jesusfreak3** [comment]
@OzzieOrca ok hope its fixed now.. I cheated, added small padding at bottom so that the background wouldnt interfere which allowed me to remove the overflow hidden causing the problem.

File: app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html [hunk 3]
```diff
@@ -1,29 +1,32 @@
-<div class="header flex justify-between">
-    <div class="self-end">
-        <div class="survey-title pv3 ph2">
-            <a ui-sref="app.ministries.ministry.surveys({ surveyId: survey.id })">
-                <img class="db mr1" ng-src="{{$ctrl.chevronLeftIcon}}"/>
```
[the remark this answers is not in the document]
Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html]: weird, when i looked, i saw a png which is why i didnt use the fill color...  IF its an svg then yah that would be the right thing to do.

[the remark this answers is not in the document]
Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html]: I can fix those others quickly, i didnt notice those when fixing the UI.  I updated other places I saw.

**Pull request merged**